packages/gatsby-theme-project-portal/src/layouts/AboutPageLayout.tsx

```diff
@@ -54,11 +54,7 @@ export const AboutPageLayout: FunctionComponent<AboutProps> = ({
   },
 }: AboutProps) => {
   return (
-    <Layout
-      activePage="About"
-      title="About"
-      description={`About the ${siteTitle}`}
-    >
+    <>
```

Comment:
There are a couple of the import statements that can be cleaned up, including `Layout`

Reply:
Good catch, thanks!